creator-node/test/audiusUsers.js

```diff
@@ -1,15 +1,33 @@
 const request = require('supertest')
+const assert = require('assert')
+const sinon = require('sinon')
+const path = require('path')
+const fs = require('fs')
 
+const models = require('../src/models')
+const ipfsClient = require('../src/ipfsClient')
+const config = require('../src/config')
 const BlacklistManager = require('../src/blacklistManager')
 
 const { getApp } = require('./lib/app')
 const { createStarterCNodeUser } = require('./lib/dataSeeds')
 const { getIPFSMock } = require('./lib/ipfsMock')
 const { getLibsMock } = require('./lib/libsMock')
+const { sortKeys } = require('../src/apiHelpers')
 
 describe('test AudiusUsers', function () {
   let app, server, session, ipfsMock, libsMock
 
+  // Will need a '.' in front of storagePath to look at current dir
```

Comment:
can we not just set the correct absolute path here instead of this workaround to get a relative path from working dir?

Reply:
yes, but my thinking was that instead of hardcoding a path, we can grab whatever value is in config in the case the path changes in the future.